src/Windowing/OpenToolkit.Windowing.Desktop/NativeWindowSettings.cs

```diff
@@ -66,6 +71,26 @@ public NativeWindowSettings()
         /// <inheritdoc />
         public string ClipboardString { get; set; }
 
+        /// <summary>
+        /// Gets or sets a value representing the current graphics API.
+        /// </summary>
+        public ContextAPI API { get; set; }
+
+        /// <summary>
+        /// Gets or sets a value representing the current graphics API profile.
+        /// </summary>
+        public ContextProfile Profile { get; set; }
+
+        /// <summary>
+        /// Gets or sets a value representing the current graphics profile flags.
+        /// </summary>
+        public ContextFlags Flags { get; set; }
+
+        /// <summary>
+        /// Gets or sets a value representing the current version of the graphics API.
+        /// </summary>
+        public Version APIVersion { get; set; }
```

Comment:
We need to add a bit more here on which versions are suggested:
```
/// Version 3.3 is chosen by default and runs on almost any hardware from the last 10 years. This will run on Windows, MacOS and Linux computers.
/// Version 4.1 is suggested for modern apps that wish to run on recent Windows, MacOS and Linux computers.
/// Version 4.6 is suggested for modern apps that wish to run only on Windows and Linux computers.
/// Note that when using a different ContextAPI is chosen, this version will need to be changed as appropriate.

Reply:
^ <remarks>